src/modules/services/submodule/mainnetSwap/submodule2/bsc/views/ViewServicesMainnetSwapBSCToSirius.vue

```diff
@@ -208,9 +208,10 @@ export default {
     const balance = ref(0);
     const coinBalance = ref(0);
     const tokenAddress = '0x2fE636d897A2a52bBc75Dc2BdE6B2FabC2359DEF';
-    const custodian = '0xd1C7BD89165f4c82e95720574e327fa2248F9cf2';
+    const custodian = '0x6A608260b6e25527AF82Be8cd12d4352145228E2';
```

Comment:
can this be set as an .env variable?

Reply:
Yup, will move the variables over after this